fabric-datafixer-v1/src/main/java/net/fabricmc/fabric/impl/datafixer/DataFixerConstants.java

```diff
@@ -0,0 +1,11 @@
+package net.fabricmc.fabric.impl.datafixer;
+
+/**
+ * This class contains constant DataFixer values for the DataFixer API.
+ */
+public class DataFixerConstants {
+	/**
+	 * Represents the version of the latest vanilla schema version. This should be updated whenever a new Schema version is added to Schemas
+	 */
+	public static final int VANILLA_DATAFIXER_VERSION = 1961; // 1.14.4
```

Comment:
Wait, I forgot why you can't grab this number from GameVersion

Reply:
That is because the latest schema version is 1961 and the World Data Version in SharedConstants for 1.14.4 is 1973.

Also you have to do some logic to convert schema version to right format so datafixer actually finds its vanilla schema.

Reply:
This isnt ideal at all. Fabric can run across versions, and its not something I want to be updating each week.

Reply:
It does seem the Vanilla Datafixer stores its schema versions and I could possibly just rip the latest version number out of the map and automatically use that with a reflection hack.